core/net/ipv6/uip-nd6.c

```diff
@@ -627,7 +627,8 @@ rs_input(void)
         /* If LL address changed, set neighbor state to stale */
         if(memcmp(&nd6_opt_llao[UIP_ND6_OPT_DATA_OFFSET],
             uip_ds6_nbr_get_ll(nbr), UIP_LLADDR_LEN) != 0) {
-          uip_ds6_nbr_t nbr_data = *nbr;
+          uip_ds6_nbr_t nbr_data;
+          nbr_data = *nbr;
```

Comment:
why is this needed? if this is only for a c64 warning I'd just revert this one change